### ISSUE TITLE
problem with get-nsxedge

### ISSUE THREAD

**Issue by MartinEtcheverry** [vmware/powernsx#574]
first of all , i am fairly new to powernsx .
when i put get-nsxedge , i get :
Exception calling "WriteContentTo" with "1" argument(s): "'☺', hexadecimal value 0x01, is an invalid character."
At C:\Users\user\Documents\WindowsPowerShell\Modules\PowerNSX\3.0.1125\PowerNSX.psm1:3516 char:9
+         $_xml.WriteContentTo($XmlWriter)
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DotNetMethodException

Exception calling "WriteContentTo" with "1" argument(s): "'↑', hexadecimal value 0x18, is an invalid character."
At C:\Users\user\Documents\WindowsPowerShell\Modules\PowerNSX\3.0.1125\PowerNSX.psm1:3516 char:9
+         $_xml.WriteContentTo($XmlWriter)
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DotNetMethodException

any hint?

**Comment by alagoutte** [comment]
Hi,

What your powershell release ?

**Comment by MartinEtcheverry** [comment]
PS C:\> $PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      0      10586  117

**Comment by alagoutte** [comment]
other cmdlet work ? (you are connected to the NSX ?)

**Comment by MartinEtcheverry** [comment]
hi , sorry for my lateness , yes is connecting correctly
PS C:\> Connect-NsxServer -NsxServer xxx.xxx.xxx.xxx
Using existing PowerCLI connection to xxx.xxx.xxx.xxx


Version             : 6.2.5
BuildNumber         : 4818372
Credential          : System.Management.Automation.PSCredential
Server              : xxx.xxx.xxx.xxx
Port                : 443
Protocol            : https
UriPrefix           :
ValidateCertificate : False
VIConnection        : vcenter
DebugLogging        : False
DebugLogfile        : C:\Users\admin\AppData\Local\Temp\2\PowerNSXLog-admin@xxx.xxx.xxx.xxx-2019_04_05_09_45_32.log

**Comment by alagoutte** [comment]
Ok... it is a very old release of NSX...

other cmdlet work ?

**Comment by MartinEtcheverry** [comment]
well get-nsxservice is working, sorry i was just starting with powernsx. so  i don't know much about the commands.

**Comment by alagoutte** [comment]
There is something on the Edge don't like by the API... 

You don't have a special configuration ?

Do you have multiple Edge ?

**Comment by MartinEtcheverry** [comment]
Hi, thank you for the reply!
as i know the nsx doesen't have any special config. 
the edges are pretty simple configs , only ha in some , but nothing more . 
i will create a new "clean" vm to discard the running environment. 
you have any sugestion about the versions of powershell /  powercli , to better support powernsx?

**Comment by alagoutte** [comment]
last release work for me... (but i think the issue coming from the NSX Release... 6.2.x is very old...)

**Comment by MartinEtcheverry** [comment]
yes , i know but checking the updates path ,  they gona update "this year" so i will try to check everything else :-D
wish me luck !

**Comment by MartinEtcheverry** [comment]
hi , i found this old problem , and actually today i get the same (and blindly open another problem #649), this time with other powershell version (windows/linux)
--------------------------------------------------------------------------------------------------------------------------------
windows - ps version  7.06 powercli 
PowerCLI Version
----------------
   VMware PowerCLI 12.2.0 build 17538434
---------------
Component Versions
---------------
   VMware Common PowerCLI Component 12.3 build 17838947
   VMware Cis Core PowerCLI Component PowerCLI Component 12.3 build 17839331
   VMware VimAutomation VICore Commands PowerCLI Component PowerCLI Component 12.3 build 17839688
   VMware VimAutomation Storage PowerCLI Component PowerCLI Component 12.3 build 17855705
   VMware VimAutomation Vds Commands PowerCLI Component PowerCLI Component 12.3 build 17856667
----------------------------------------------------------------------------------------------------------------------------------------------------------------------
